src/cli.ts

```diff
@@ -257,6 +267,14 @@ function configureCommonCommandOptions(_yargs: Argv) {
       default: 'v',
       description: 'The prefix used in tags before the version number.',
       type: 'string',
+    })
+    .option('versionBeforePkgRename', {
```

Comment:
See note above abbreviations. I feel like it would be more consistent with how "package" is spelled in this project if the option that this CLI took was `--version-before-package-rename` rather than `--version-before-pkg-rename`. What do you think?